content/en/docs/collector/distributions.md

```diff
@@ -22,3 +24,29 @@ functionality like
 [connectors](../building/connector). The tool used to build distributions
 [ocb](../custom-collector) (OpenTelemetry Collector Builder) is available to
 build your own distributions.
+
+## Vendor Distributions
```

Comment:
Perhaps the heading should read "Third-party" to match the terminology used in the Ecosystem distributions page.